.azure/pr-check.yaml

```diff
@@ -23,5 +30,39 @@ steps:
       python .azure/scripts/validate_samples.py
     displayName: "Metadata Validation"
 
+  # Give execute permissions to trivy-scan.sh
+  - script: |
+      echo "Giving execute permission to trivy-scan.sh"
+      chmod +x .azure/scripts/trivy-scan.sh
+    displayName: "Set execute permissions"
+
+  - template: install/install-trivy.yaml@common-templates
+  - script: |
+        echo "##[group]Running Trivy scan for each image"
+
+        # Initialize exit_code as 0 (success)
+        exit_code=0
+
+        # Read image_urls.txt and pass each image to trivy-scan.sh
+        while IFS= read -r image; do
```

Comment:
Do we need `IFS= ` here? Are we expecting whitespace in the image URLs?

Reply:
Image urls dont have whitespaces so we can remove IFS.